src/client/app.coffee

```diff
@@ -128,7 +44,7 @@ class Arena
             randomPoint = @map.randomPoint()
             randomTeam = _.sample((name for name, r of @teams))
 
-            @focusedUIPlayer = new UIPlayer @, @handler, randomPoint, randomTeam
+            @focusedUIPlayer = new UIPlayer this, @handler, randomPoint, randomTeam
```

Comment:
Fair enough, I prefer @. I think they prefer this as @method implies that @ includes the dot accessor... but to me @ just feels more coffeescripty